runtime/src/accounts_db.rs

```diff
@@ -1330,10 +1330,8 @@ impl AccountsDB {
         hasher.result()
     }
 
-    pub fn include_owner_in_hash(slot: Slot) -> bool {
```

Comment:
@mvines Well, I've found we can't remove this gating until the cluster is over an epoch after eager rent collection is enabled.

That's because this `slot` is the slot a given account is updated at, not the slot when the bank hash is calculated for the bank at.... And accounts are at various slots. And accounts last updated older than 14_000_000 must be calculated with old hashing method (ie, don't include owner).

So we must wait to remove this for tds until eager rent collection updates all accounts...


Maybe that's reason #10163 is occurring too; we need this logic for master to consume tds snapshots.

Reply:
Ah, ok cool.  Happy to let this PR sit until Monday.  I was just trying to get ahead of things for the v1.0 -> v1.1 mainnet-beta upgrade

Reply:
@mvines See https://github.com/solana-labs/solana/issues/10163#issuecomment-632604747.

Sadly, we can't land this as-is....

I can take care of this. :)

Reply:
Oh sure.  The trick is that we're starting to run out of time.  Here's what the release timeline looks like for the next week:
1.  master branch forks into the v1.2 branch today
2. release v1.2.0 probably on Tuesday the 26th
3. upgrade devnet to v1.2.0 ideally 
4. by the 29th, upgrade mainnet-beta to v1.1
4. by the 29th, upgrade testnet to v1.2

Reply:
Ok, I'll adjust the transition plan accordingly. :)

Reply:
I filed https://github.com/solana-labs/solana/issues/10227 to track this change for a little later